packages/terra-alert/src/Alert.jsx

```diff
@@ -171,8 +171,7 @@ const Alert = ({
 
   return (
     <ResponsiveElement
-      responsiveTo="parent"
```

Comment:
did you intend to remove this?

Reply:
Yes, parent is the default value of responsiveTo, so it is redundant have in code